### PR TITLE
chore: make web config runtime config

### PR DESCRIPTION
**Pull request by sborrazas**

### CODE REVIEW

File: config/dev.exs [hunk 1]
```diff
@@ -1,33 +1,12 @@
...
 config :ae_mdw, :enable_livedashboard, true
 
 config :ae_mdw, TelemetryMetricsStatsd,
   host: "localhost",
   port: 8125
 
-# Endpoints
-protocol_opts = [max_request_line_length: 1_024, max_skip_body_length: 1_024]
-
-config :ae_mdw, AeMdwWeb.Endpoint,
```
Comment (jyeshe) [inline config/dev.exs:16]: Some of these are needed in dev, used differently from prod. I see there is a boolean flag on `debug_errors` in the `runtime.exs`. Could we add the same for `live_view`? Or just leave the endpoint declaration here in `dev.exs`.

File: config/runtime.exs [hunk 2]
```diff
@@ -2,13 +2,39 @@ import Config
 
 env = config_env()
 
+# Chain
+config :aecore, network_id: System.get_env("NETWORK_ID", "ae_mainnet")
```
Comment (jyeshe) [inline config/runtime.exs:6]: This is dev only without docker, keeping it in `dev.exs` would avoid conflicts in prod.

File: docker-compose-test.yml [hunk 3]
```diff
@@ -14,4 +14,6 @@ services:
     volumes:
       - ${PWD}:/app
     environment:
-      - AETERNITY_CONFIG=/home/aeternity/aeternity.yaml
+      AETERNITY_CONFIG: /home/aeternity/aeternity.yaml
+      PORT: 4002
+      WS_PORT: 4003
```
Comment (jyeshe) [inline docker-compose-test.yml:19]: Same as first comment.

File: .github/workflows/test-action.yml [hunk 4]
```diff
@@ -43,6 +43,9 @@ jobs:
 
       - name: Execute coverage
         run: elixir --sname aeternity@localhost -S mix coveralls
+        env:
+          PORT: 4002
+          WS_PORT: 4003
```
Comment (jyeshe) [inline .github/workflows/test-action.yml:48]: This is not necessary because tests don't use the http server and ws is only accessed by test cases.

**Review by jyeshe — CHANGES_REQUESTED** [review]
More flexibility for other setups, nice! Nothing operational that affects prod only some chore comments.

File: config/runtime.exs (hunk 2 above)
Comment (sborrazas) [reply config/runtime.exs:6]: Move this back to dev.config. Thanks

File: .github/workflows/test-action.yml (hunk 4 above)
Comment (sborrazas) [reply .github/workflows/test-action.yml:48]: If I don't add them, these tests fail: https://github.com/aeternity/ae_mdw/blob/master/test/ae_mdw/websocket/socket_handler_test.exs#L10

File: config/dev.exs (hunk 1 above)
Comment (sborrazas) [reply config/dev.exs:16]: Back to dev too

**Comment by sborrazas** [comment]
> All good for me except that test ports could be setup in a single file.

moved compile-time config for tests back to test.exs

**Review by jyeshe — APPROVED** [review]

**Pull request merged**